docs/topics/api-guidelines-build-for-multiplatform.md

```diff
@@ -7,6 +7,15 @@ The following sections provide guidelines to help you build a Kotlin Multiplatfo
 
 ## Maximize your reach
 
+> To streamline artifact production, you can try out an [experimental](components-stability.md#stability-levels-explained) feature for publishing Kotlin Multiplatform libraries from any host using cross-compilation.
```

Comment:
Another way to simplify this would be to remove the components-stability link and not have a separate sentence for "For more information" (where we already state that it's Experimental and how to enable it anyway)

> To streamline artifact production, you can try out Experimental [cross-compilation](multiplatform-publish-lib.md#host-requirements) to publish Kotlin Multiplatform libraries from any host using cross-compilation.

Reply:
Thank you for your suggestion! 
I updated the text (removed the experimental link and placed the link on cross-compilation) and tried all the options. 
I think the best way is to follow your second suggestion is to place it as the third paragraph to make the text feel a bit less busy! 👍